assets/triangle_two.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" data-name="Layer 1" viewBox="0 0 100 125" x="0px" y="0px"><title>tri 5</title><path d="M51.94,86.74,14.14,21.27l-5.5-9.53L7.35,14h86.6l-1.3-2.26L54.85,77.21l-5.5,9.53a1.5,1.5,0,0,0,2.59,1.51l37.8-65.47,5.5-9.53A1.51,1.51,0,0,0,93.95,11H7.35A1.51,1.51,0,0,0,6,13.26l37.8,65.47,5.5,9.53A1.5,1.5,0,0,0,51.94,86.74Z"/><path d="M6.53,68.11l76,.31,10.9,0a1.5,1.5,0,0,0,0-3l-76-.31-10.9,0a1.5,1.5,0,0,0,0,3Z"/><text x="0" y="115" fill="#000000" font-size="5px" font-weight="bold" font-family="'Helvetica Neue', Helvetica, Arial-Unicode, Arial, Sans-serif">Created by Ayse Muskara</text><text x="0" y="120" fill="#000000" font-size="5px" font-weight="bold" font-family="'Helvetica Neue', Helvetica, Arial-Unicode, Arial, Sans-serif">from the Noun Project</text></svg>
```

Comment:
It sounded like you wanted to credit the artist for your images. I think it's fine to put this at the bottom of the image. 
Crediting the artist is something you should definitely do. And also ask for permission to use the art if permission is not explicitly given. It's the right thing to do, and it wil save you from copyright issues. Or, use art from the public domain.